### PR TITLE
coil.com removed

### PR DESCRIPTION
**Pull request by timwolf1**

### CODE REVIEW

**Comment by akaye47** [comment]
KANYE WEST•

@kanyewest

Ima fix wolves

RETWEETS

21,233

LIKES

44,286

Hi TimmA!

(My editor makes me put that copyright shit.) How are you?! What are you up to since the Chief retired?! HMU!!

Then she makes my pic a thumbnail. Enjoy your freedom, friend! I've been a hostage for almost, well, depends on what you consider a threat to National security atm. 🙄

I miss your face!

❤️
A


________________________________
From: timwolf1 ***@***.***>
Sent: Wednesday, April 12, 2023 10:17 AM
To: sherlock-project/sherlock ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sherlock-project/sherlock] coil.com removed (PR #1771)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/sherlock-project/sherlock/pull/1771

Commit Summary

  *   b54d6aa<https://github.com/sherlock-project/sherlock/pull/1771/commits/b54d6aaf94ef2a4ffffea1906be664b8ac40e7fc> coil.com removed

File Changes

(4 files<https://github.com/sherlock-project/sherlock/pull/1771/files>)

  *   M removed_sites.json<https://github.com/sherlock-project/sherlock/pull/1771/files#diff-abc25f6793cbab2004e2268a2663c6235b6babeb7722857d3f3bdae778f4c0f4> (16)
  *   M removed_sites.md<https://github.com/sherlock-project/sherlock/pull/1771/files#diff-a92204a301dd9bfeb48620d08a5fe4eeec2e66b8526caa6ff609a71e1faa7e71> (23)
  *   M sherlock/resources/data.json<https://github.com/sherlock-project/sherlock/pull/1771/files#diff-8a6ed70a228044b511cdffa4ca8bbefffd74763903a04dc380ad74f673ed8e02> (16)
  *   M sites.md<https://github.com/sherlock-project/sherlock/pull/1771/files#diff-30c0c111452d67a760d4787ee4ad412d09aea675fb8ce43cc83ba5fdcb9bed23> (1)

Patch Links:

  *   https://github.com/sherlock-project/sherlock/pull/1771.patch
  *   https://github.com/sherlock-project/sherlock/pull/1771.diff

**Comment by timwolf1** [comment]
@akaye47 i am deeply confused

**Comment by sdushantha** [comment]
Thank you for catching this and fixing the issue!

File: sherlock/resources/data.json [hunk 1]
```diff
@@ -489,22 +489,6 @@
...
-        "userShortName": "{}"
-      }
-    },
-    "url": "https://coil.com/u/{}",
-    "urlMain": "https://coil.com/",
-    "urlProbe": "https://coil.com/gateway",
-    "username_claimed": "adam"
-  },
   "Coinvote": {
     "errorType": "status_code",
```
Comment (Fridaygaruba) [inline sherlock/resources/data.json:493]: https://www.facebook.com/profile.php?id=100090527732263

**Review by Fridaygaruba — COMMENTED** [review]
https://www.facebook.com/profile.php?id=100090527732263

**Pull request merged**